TriblerGUI/widgets/trustpage/js/style_config.js

```diff
@@ -18,6 +18,7 @@ var config = {
         highlightOutDelay: 300
     },
     node: {
+        userLabelText: "Me",
```

Comment:
I'm still not completely sold on the "Me" label. Maybe change the appearance of the entire node instead of only the label